crates/messages/Cargo.toml

```diff
@@ -1,24 +1,41 @@
 [package]
 name = "sqd-messages"
 license = "AGPL-3.0-or-later"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
It should be `2.1.0` because new functionality is added, but I suggest we keep it `2.0.0` until merging to main. Commit refs are used everywhere until then anyway